syntax = "proto2";

package proto.db;

message call_command {
	required string sql = 1;
	repeated bytes args = 2;
}